content/providers/01-ai-sdk-providers/00-ai-gateway.mdx

```diff
@@ -134,7 +134,27 @@ const gateway = createGateway({
 
 ### OIDC Authentication (Vercel Deployments)
 
-When deployed on Vercel, the AI Gateway provider automatically uses OIDC tokens for authentication without requiring an API key.
+When deployed to Vercel, the AI Gateway provider supports authenticating using OIDC (OpenID Connect)
+tokens without API Keys.
+
+#### How OIDC Authentication Works
+
+1. **In Production/Preview Deployments**:
+
+   - OIDC authentication is automatically handled
+   - No manual configuration needed
+   - Tokens are automatically obtained and refreshed every 12 hours
+
+2. **In Local Development**:
+   - Use the [Vercel CLI](https://vercel.com/docs/cli) to run `vercel env pull` which fetches the token
+   - Since OIDC tokens expire after 12 hours, you'll need to run `vercel env pull` again to refresh them
```

Comment:
an additional note - if the developer uses `vercel dev` to run the dev server, it will auto-refresh the token for them